missilewars-plugin/src/main/java/de/butzlabben/missilewars/game/Game.java

```diff
@@ -623,7 +646,6 @@ public void spawnFireball(Player player, ItemStack itemStack) {
     }
 
     public void setArena(Arena arena) {
-        Preconditions.checkNotNull(arena);
```

Comment:
I'm not sure why we need that too.

Reply:
IntelliJ sometimes complains, that things could be null and suggests to add these. I have added these a long time ago, but they aren't necessary.